packages/webpack-extraction-plugin/src/webpackLoader.ts

```diff
@@ -78,11 +75,41 @@ function webpackLoader(
   }
 
   if (result) {
-    if (result.cssRules) {
+    if (result.cssRulesByBucket) {
+      const css = Object.entries(result.cssRulesByBucket).reduce((acc, [cssBucketName, cssBucketRules]) => {
+        if (cssBucketName === 'm') {
+          return (
+            acc +
+            cssBucketRules
+              .map(entry => {
+                return (
+                  `/** @griffel:css-start [${cssBucketName}] [${JSON.stringify(entry[1])}] **/` +
+                  '\n' +
```

Comment:
nit: would look g abit nicer if you used a string array and then joined with newline

Reply:
Agree, pushed